stingray/lightcurve.py

```diff
@@ -1059,11 +1059,15 @@ def plot(self, witherrors=False, labels=None, axis=None, title=None,
         if title is not None:
             plt.title(title)
 
-        if save:
+        if save is True:
```

Comment:
Since `save` is declared as `bool` in the docstring, and it is a very minor detail, I would personally avoid complicating the code with a check that it is boolean.
I would just use 
```
if save:
    (...)
else:
    (...)
```

Reply:
I did it so because if the user runs `a.plot(save="a string")`, then it saves the plot in a file (which doesn't make sense). If you say so, I'll change it. What do you say?

Reply:
It's just that there are a thousand different worse ways to break the code by declaring a variable with the wrong type, this looks like a very minor case. Also, the user should follow the docstring. 
I expect that sooner or later we will start using type hints (PEP 484), when the implementation is more robust and it doesn't fail with numpy types.

Reply:
Fixed it. :+1: